cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition/config.jelly

```diff
@@ -25,10 +25,14 @@
 
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler">
   <f:entry title="${%Script}" field="script">
-    <f:textarea checkMethod="post"/>
+    <div id="workflow-editor-wrapper" style="display: none; position: relative">
+        <f:textarea checkMethod="post"/>
+        <div id="workflow-editor" class="pane-frame" style="height: 250px;"/>
+    </div>
   </f:entry>
   <f:entry field="sandbox">
     <f:checkbox title="${%Use Groovy Sandbox}" default="${!h.hasPermission(app.RUN_SCRIPTS)}" />
   </f:entry>
   <st:include it="${descriptor.snippetizer}" page="block.jelly"/>
+  <st:adjunct includes="org.jenkinsci.plugins.workflow.workflow"/>
```

Comment:
Perhaps I am missing something, but which file is this referring to? I see no `src/main/resources/org/jenkinsci/plugins/workflow/workflow.js` or `…/workflow.css`.

Also it seems like it should have a more distinctive name, since it presumably applies only to `CpsFlowDefinition`.